maestro-android/src/androidTest/java/dev/mobile/maestro/MaestroDriverService.kt

```diff
@@ -99,86 +99,98 @@ class Service(
         request: MaestroAndroid.LaunchAppRequest,
         responseObserver: StreamObserver<MaestroAndroid.LaunchAppResponse>
     ) {
-        val context = InstrumentationRegistry.getInstrumentation().targetContext
+        try {
+            val context = InstrumentationRegistry.getInstrumentation().targetContext
 
-        val intent = context.packageManager.getLaunchIntentForPackage(request.packageName)
+            val intent = context.packageManager.getLaunchIntentForPackage(request.packageName)
 
-        if (intent == null) {
-            Log.e("Maestro", "No launcher intent found for package ${request.packageName}")
-            responseObserver.onError(RuntimeException("No launcher intent found for package ${request.packageName}"))
-            return
-        }
+            if (intent == null) {
+                Log.e("Maestro", "No launcher intent found for package ${request.packageName}")
+                responseObserver.onError(RuntimeException("No launcher intent found for package ${request.packageName}"))
```

Comment:
You need to build the APKs I see the APKs are not changed

Reply:
How do I do this? 🤔 Can you show me?

Reply:
Sure, ensure that you are using java 8 in your cli from root of maestro. Then run following commands:

```
./gradlew maestro-android:assembleAndroidTest
./gradlew maestro-android:assemble
```

This should be generating 2 APKs